### PR TITLE
add Recipes section under Guides category

### PR DESCRIPTION
**Pull request by fricklerhandwerk**
this follows #701 and recent discussion around reshuffling the FAQ with @wamirez and @infinisil

### CODE REVIEW

File: source/guides/index.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(recipes)=
+(guides)=
 # Recipes
```
Comment (wamirez) [inline source/guides/index.md:2]: :s/Recipes/Guides to update the sidebar

File: source/contributing/documentation/diataxis.md [hunk 2]
```diff
@@ -39,13 +39,13 @@ A concept can also describe the historical context behind why something works th
 
 If you find yourself wanting to write about the nitty gritty details of how something works, you most likely want to write an explanation.
 
-### Recipes vs. Tutorials
+### Guide vs. Tutorials
```
Comment (wamirez) [inline source/contributing/documentation/diataxis.md:42]: Maybe make Guide also plural or conversely, make Tutorial a singular (pedantic changes just to keep consistency)

File: source/guides/index.md (hunk 1 above)
Comment (fricklerhandwerk) [reply source/guides/index.md:2]:
```suggestion
# Guides
```

Thanks, good catch!

File: source/contributing/documentation/diataxis.md (hunk 2 above)
Comment (fricklerhandwerk) [reply source/contributing/documentation/diataxis.md:42]:
```suggestion
### Guides vs. Tutorials
```

Yes, correct, thanks

**Comment by fricklerhandwerk** [comment]
We may want to remove the Diataxis page eventually, as it doesn't really provide any information that's not on the official site.

**Review by wamirez — APPROVED** [review]

**Pull request merged**